Project.toml

```diff
@@ -9,10 +9,12 @@ IfElse = "615f187c-cbe4-4ef1-ba3b-2fcf58d6d173"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 SBML_jll = "bb12108a-f4ef-5f88-8ef3-0b33ff7017f1"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StructHelpers = "4093c41a-2008-41fd-82b8-e3f9d02b504f"
```

Comment:
I guess you aren't going to be happy to _add_ yet another dependency, but the problem is that `==` defaults to `===` for custom structs (see my `#gripes` today in Slack), and so we have to extend all `==` methods for all the structures we want to compare, but this is error prone as we may add new fields to these structs.  `StructHelpers` does this for us automatically.

Reply:
I guess this is good, literally adds just Structhelpers and ConstructionBase (transitively), both of which are tiny

Reply:
(better than writing == instances for everything)